### DISCUSSION TITLE
best way to set no proxy for domains or subdomains

### DISCUSSION THREAD

**Discussion by mohammadRezaeian** [FranzKafkaYu/x-ui#253] [Q&A] [answered]
hi thank you for create this project
i need to know is it possible set no proxy for all domains like:ir
or another domains (use panel or use set configuration in x-ray )?

**Comment by Jabi-james** [comment, marked as answer]
> hi thank you for create this project i need to know is it possible set no proxy for all domains like:ir or another domains (use panel or use set configuration in x-ray )?

You can not make a “Direct” connection for specific domains from server or xray setting on panel to bypass proxy for example “ir” domains or any domains.. It should be done on the client side (V2rayNG or etc.).. Although you can block specific domains on the server to avoid discovering your  server IP by the GFW..
